mllib/src/main/scala/org/apache/spark/ml/tree/impl/DecisionTreeMetadata.scala

```diff
@@ -42,6 +42,7 @@ import org.apache.spark.rdd.RDD
 private[spark] class DecisionTreeMetadata(
     val numFeatures: Int,
     val numExamples: Long,
+    val weightedNumExamples: Double,
```

Comment:
shouldn't the new params weightedNumExamples and  minWeightFractionPerNode be added to the documentation for this method?